dotnet-authserver/src/TeacherIdentity.AuthServer/Pages/Admin/User.cshtml

```diff
@@ -47,9 +47,16 @@
                     <govuk-summary-list-row-action asp-page="EditUserEmail" asp-route-userId="@Model.UserId" visually-hidden-text="email address">Change</govuk-summary-list-row-action>
                 </govuk-summary-list-row-actions>
             </govuk-summary-list-row>
+            <govuk-summary-list-row>
+                <govuk-summary-list-row-key>Mobile number</govuk-summary-list-row-key>
+                <govuk-summary-list-row-value class="empty-hyphens">@Model.MobileNumber</govuk-summary-list-row-value>
```

Comment:
Don't  need the class here (it's only used for email addresses)
```suggestion
                <govuk-summary-list-row-value>@Model.MobileNumber</govuk-summary-list-row-value>
```

Reply:
removed `class="empty-hyphens"`